Packs/Mimecast/ReleaseNotes/2_3_0.md

```diff
@@ -0,0 +1,22 @@
+#### Integrations
+
+##### Mimecast v2
+
+- Added support for **API 2.0** with **client id** and **secret id** parameters.
+- Deprecated the command ***mimecast-update-policy***. Use ***mimecast-update-block-sender-policy*** instead.
+- Deprecated the command ***mimecast-create-policy***. Use ***mimecast-create-block-sender-policy*** instead.
+- Deprecated the command ***mimecast-list-blocked-sender-policies***. Use ***mimecast-list-policies*** instead.
+- Added support for additional types of policies in the command ***mimecast-get-policy***.
+- Added support for additional types of policies in the command ***mimecast-delete-policy***.
+
```

Comment:
- Add the following commands:

Reply:
It falls in Bild:
"is not using one of our templates, consider changing it to fit our standard."

is that ok?
Updated the integration to add the following commands: